great_expectations/datasource/batch_kwargs_generator/batch_kwargs_generator.py

```diff
@@ -202,7 +201,7 @@ def _build_batch_kwargs(self, batch_parameters):
         raise NotImplementedError
 
     # TODO: deprecate generator_asset argument
-    def yield_batch_kwargs(self, generator_asset=None, data_asset_name=None, **kwargs):
+    def yield_batch_kwargs(self, data_asset_name=None, generator_asset=None, **kwargs):
```

Comment:
Isn't this a breaking change?

`yield_batch_kwargs("foo", "bar")`
`yield_batch_kwargs("bar", "foo")`

Reply:
In this case, no, because one cannot provide both generator_asset and data_asset_name (already)...essentially you have to be using kwargs for anything beyond the first parameter, and so this just allows you to use the first parameter without getting a (needless) deprecation warning.